### ISSUE TITLE
Random Disconnects

### ISSUE THREAD

**Issue by sayig18** [rivafarabi/deckboard#108]
My tablet will connect and work for about 15 minutes before disconnecting saying "Uh-oh!". I have to press okay and it reconnects again but this happens very often. Already checked my firewall. Don't know what else to do.

**Comment by pythargames** [comment]
I also have this problem & have gone through all the troubleshooting.

**Comment by Teraldar** [comment]
Same problem here.

**Comment by rZiMiZ** [comment]
Yes, me too :( i use this software for yeras, with the pro, and always this problem :(
